src/components/member-profile/active-task/index.js

```diff
@@ -21,6 +22,20 @@ const ActiveTask = ({ taskDetails }) => {
   );
   const showProgressIndicator = progressIndicator(showEstimatedDay, classNames);
 
+  const handleProgressBarColor = () => {
+    const colorOptions = [];
+    if (percentageOfDaysRemaining >= percentOfTaskLeft) {
+      colorOptions.push(classNames.progressIndicatorGreen); // green
+    } else if (percentageOfDaysRemaining < 50 && percentOfTaskLeft > 75) {
+      colorOptions.push(classNames.progressIndicatorOrange); // orange
+    } else if (percentageOfDaysRemaining < 25 && percentOfTaskLeft > 35) {
+      colorOptions.push(classNames.progressIndicatorRed); // red
+    } else {
+      colorOptions.push(classNames.progressIndicatorYellow); // yellow
+    }
+    return colorOptions;
+  };
```

Comment:
This can be refactored as follows:

```js
  if (percentageOfDaysRemaining >= percentOfTaskLeft) {
      return classNames.progressIndicatorGreen; // green
  }
  if (percentageOfDaysRemaining < 50 && percentOfTaskLeft > 75) {
      return classNames.progressIndicatorGreen // green
   }
  ... and so on, and you just need to call the function to get the className value
```

Reply:
I have made changes to the code and refactored as mentioned.